### PR TITLE
fix(t34): fix edit map and map pool bug

### PR DESCRIPTION
**Pull request by AlekseySamokhin**

### CODE REVIEW

File: pages/maps/[id].tsx [hunk 1]
```diff
@@ -1,29 +1,59 @@
...
-import { UPDATE_MAP } from '@/modules/maps/api/maps/mutations';
-import { UpdateMapMutation, UpdateMapMutationVariables } from '@/src/generated/graphql';
+import { GET_MAP, UPDATE_MAP } from '@/modules/maps/api/maps';
+
+import type {
+  UpdateMapMutation,
+  UpdateMapMutationVariables,
+} from '@/src/generated/graphql';
+
+type ITypeDataForm = {
```
Comment (killmenot) [inline pages/maps/[id].tsx:18]: @AlekseySamokhin

1. ITypeFormValues
2. extract Form from page

**Review by killmenot — APPROVED** [review]

**Comment by killmenot** [comment]
Please fix notes, then u can merge

File: pages/map-pools/[id].tsx [hunk 2]
```diff
@@ -1,33 +1,49 @@
...
+  UpdateMapPoolMutationVariables,
+} from '@/src/generated/graphql';
 
+const MapPoolDetail = () => {
+  const [valueForm, setValueForm] = useState<string>('');
   const router = useRouter();
-
   const { id } = router.query;
 
+  const { data } = useQuery(GET_MAP_POOL, {
```
[the remark this answers is not in the document]
Comment (AlekseySamokhin) [reply pages/map-pools/[id].tsx:23]: @killmenot I didn't find the right type

[the remark this answers is not in the document]
Comment (killmenot) [reply pages/map-pools/[id].tsx:23]: Review what is generated, it could be useful

[the remark this answers is not in the document]
Comment (AlekseySamokhin) [reply pages/map-pools/[id].tsx:23]: I looked at the folder and tried different versions but it doesn't work.

Error -> 
'GetMapPoolsDocument' refers to a value, but is being used as a type here. Did you mean 'typeof GetMapPoolsDocument'?ts(2749)

[the remark this answers is not in the document]
Comment (AlekseySamokhin) [reply pages/map-pools/[id].tsx:23]: Understood, thanks!

**Pull request merged**